### ISSUE TITLE
Godot 4 support

### ISSUE THREAD

**Issue by jamie-pate** [jamie-pate/godot-code-coverage#1]
Currently this tool is known to support godot 3.5.1 only.

**Comment by joshromoff** [comment]
hi is 4.0 supported yet? thanks!

**Comment by jamie-pate** [comment]
Yes, the main branch is working on 4.2.2 for me.. starting with https://github.com/jamie-pate/godot-code-coverage/commit/f514e3322b1457e4ba5d55cdb1b484ebe22c5c9f

It works with my current code base, but it's not been tested on any others :crossed_fingers:
